templates/add-printer.tmpl

```diff
@@ -32,7 +32,7 @@
 </TR>
 <TR>
 <TH CLASS="label">Sharing:</TH>
-<TD><INPUT TYPE="CHECKBOX" NAME="PRINTER_IS_SHARED" ID="PRINTER_IS_SHARED" {PRINTER_IS_SHARED=1?CHECKED:}>
+<TD><INPUT TYPE="CHECKBOX" NAME="PRINTER_IS_SHARED" ID="PRINTER_IS_SHARED" VALUE="CHECKED" {PRINTER_IS_SHARED=1?CHECKED:}>
```

Comment:
The "VALUE" here is not valid.

Reply:
@michaelrsweet Why isn't it valid? It looks like the mean how to change what is returned in form if the checkbox is checked based on this page https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox .

Additionally, we save the same value into the checkbox variables in ```do_menu()``` if the option is on, so IMHO we should get the same value from the form if user checks the option. If the value is not set for checkbox input type, the incoming value will always be "ON", so we would have to adjust admin CGI script accordingly if we don't use VALUE attribute.

Reply:
Sorry, I misremembered the current HTML spec WRT checkboxes. But since the default value is CHECKED and we want it to return CHECKED, I don't see a reason to add the VALUE everywhere there is a checkbox.

Reply:
@michaelrsweet  If there is no VALUE attribute (meaning VALUE="smth"), the form will contain NAME="ON" if the checkbox is checked, so in our case here PRINTER_IS_SHARED="ON". IIUC ```{PRINTER_IS_SHARED=1?CHECKED:}``` decides only what will be showed when the page is loaded, but not what will be sent in form if the box stays checked.

Reply:
Sigh, ok well then we either add the value attribute or allow any value to be treated as checked in `cgiGetCheckbox`.

Reply:
I would add the value attribute into the templates - we already set the checkbox variables in CGI to "checked" already before this change, so it can be consistent.

Reply:
@michaelrsweet so I reverted the 1d -> 86400 change, otherwise the changes are the same since it already contained value attribute additions. Ready for review.